packages/cozy-stack-client/src/PermissionCollection.spec.js

```diff
@@ -21,7 +21,30 @@ describe('PermissionCollection', () => {
       client.fetch.mockReset()
       client.fetchJSON.mockReturnValue(Promise.resolve({ data: [] }))
     })
+    describe('create', () => {
+      it('calls with the right agrs', async () => {
```

Comment:
maybe something more explicit like `should be called with 'code' argument` ?

Anyway there is a typo on 'agrs'